set.mm

```diff
@@ -529430,2958 +529515,2965 @@ standard proof of the implication in modal logic (B5 ` => ` 4).  Its dual
+       require ~ ax-10 , ~ ax-11 , ~ ax-12 , ~ ax-13 .  Instance of ~ nfbidv .
+       Note that the version of ~ axc15 with a disjoint variable condition is
+       actually ~ ax12v2 (up to adding a superfluous antecedent).  (Contributed
+       by BJ, 17-Jun-2019.)  (Proof modification is discouraged.) $)
+    bj-drnf2v $p |- ( A. x x = y -> ( F/ z ph <-> F/ z ps ) ) $=
+      ( weq wal nfbidv ) CDGCHABEFI $.
   $}
 
-  $( Uniqueness is equivalent to existence being equivalent to unique
-     existence.  (Contributed by BJ, 14-Oct-2022.) $)
-  bj-moeub $p |- ( E* x ph <-> ( E. x ph <-> E! x ph ) ) $=
-    ( wmo wex weu wi wb moeu euex impbi mpi biimp impbii bitri ) ABCABDZABEZFZO
-    PGZABHQRQPOFRABIOPJKOPLMN $.
-
-  $( Obsolete proof of ~ sbidm temporarily kept here to check it gives no
-     additional insight.  (Contributed by NM, 8-Mar-1995.)
-     (Proof modification is discouraged.)  (New usage is discouraged.) $)
-  bj-sbidmOLD $p |- ( [ y / x ] [ y / x ] ph <-> [ y / x ] ph ) $=
-    ( wsb wb weq equsb2 sbequ12r sbimi ax-mp sbbi mpbi ) ABCDZAEZBCDZMBCDMECBFZ
-    BCDOBCGPNBCACBHIJMABCKL $.
-
-$(
   ${
-    $d ph ps $.
-    @( An unprovable true scheme.
-
-       It is true since the disjoint variable condition implies that ` x ` does
-       not occur in at least one of ` ph ` and ` ps ` , so at least one of the
-       two disjuncts is true.
-
-       It is not provable from any consistent axiom system containing no
-       disjoint variable condition of that form.  Indeed, such a disjoint
-       variable condition cannot be produced from disjoint variable conditions
-       of one of the other two forms, DV ` ( x , y ) ` or DV ` ( x , ph ) ` .
-       Therefore, if it were provable, then so would the same scheme without
-       this disjoint variable condition be, but that scheme is obviously false
-       (take ` ph ` and ` ps ` to be ` x = y ` for instance).
-
-       See also ~ ax-bj-00 .
-
-       Megill's completeness theorem concerns schemes with disjoint variable
-       conditions of the form DV ` ( x , y ) ` or DV ` ( x , ph ) ` .  The
-       present scheme shows that it does not generalize to include schemes that
-       contain disjoint variable conditions of the form DV ` ( ph , ps ) ` .
-
-       (Contributed by BJ, 19-Mar-2021.)  (New usage is discouraged.) @)
-    ax-bj-0 $a |- ( ( ph -> A. x ph ) \/ ( ps -> A. x ps ) ) $.
-
-    @( A variant of ~ ax-bj-0 , morally stronger, with the advantage of binding
-       every possible occurrence of ` x ` in any of its instances (see for
-       instance ~ df-nf and ~ df-bj-nf ).
-
-       (Contributed by BJ, 19-Mar-2021.)  (New usage is discouraged.) @)
-    ax-bj-00 $a |- ( ( E. x ph -> A. x ph ) \/ ( E. x ps -> A. x ps ) ) $.
+    $d x y $.
+    bj-equs45fv.1 $e |- F/ y ph $.
+    $( Version of ~ equs45f with a disjoint variable condition, which does not
+       require ~ ax-13 .  Note that the version of ~ equs5 with a disjoint
```

Comment:
@benjub I am a bit late, but [bj-equs45fv](https://us.metamath.org/mpeuni/bj-equs45fv.html) does depend on ax-13 because it uses [equs5a](https://us.metamath.org/mpeuni/equs5a.html). There is [equs5aALT](https://us.metamath.org/mpeuni/equs5aALT.html) as equivalent version of [equs5a](https://us.metamath.org/mpeuni/equs5a.html) that does not use ax-13, but I'm not sure if it's ok to use it since ALT usage is generally discouraged.

Reply:
So if I understand correctly this can be solved by adding a new equs5av and use it instead of equs5a:

<pre>
  ${
    $d x y $.
    $( Version of ~ equs5a with a dv condition, which does not require ~ ax-13 $)
    equs5av $p |- ( E. x ( x = y /\ A. y ph ) -> A. x ( x = y -> ph ) ) $=
      ( weq wal wa wi nfa1 ax12v2 spsd imp exlimi ) BCDZACEZFMAGZBEZBOBHMNPMAPCAB
      CIJKL $.
  $}
</pre>

<pre>
  ${
    $d x y $.
    bj-equs45fv.1 $e |- F/ y ph $.
    $( Version of ~ equs45f with a disjoint variable condition, which does not
       require ~ ax-13 .  Note that the version of ~ equs5 with a disjoint
       variable condition is actually ~ sb56 (up to adding a superfluous
       antecedent).  (Contributed by BJ, 11-Sep-2019.)
       (Proof modification is discouraged.) $)
    bj-equs45fv $p |- ( E. x ( x = y /\ ph ) <-> A. x ( x = y -> ph ) ) $=
      ( weq wa wex wi wal nf5ri anim2i eximi equs5av syl equs4v impbii ) BCEZAFZ
      BGZQAHBIZSQACIZFZBGTRUBBAUAQACDJKLABCMNABCOP $.
  $}
</pre>

Removing ax-13 from bj-equs45fv this way seems to work:
<pre>
MM> SHOW TRACE_BACK bj-equs45fv / AXIOMS / MATCH ax-*
Statement "bj-equs45fv" assumes the following axioms ($a statements):
  ax-mp ax-1 ax-2 ax-3 ax-gen ax-4 ax-5 ax-6 ax-7 ax-10 ax-12
</pre>

Reply:
A `git checkout 561248a805a60cd9eedef4d31b6ccc1447b07798` showed that at the time, ~equs5a had the proof of the current ~euqs5aALT (so probably the proof was changed and an *ALT version kept, which I think is a good thing).  I think that indeed you could contribute ~equs5av.  Then, could you please update bj-equs45fv by replacing ~equs5a with your ~equs5av ?

Reply:
Ok, I made the PR: https://github.com/metamath/set.mm/pull/3697